application/cxxopts.hpp

```diff
@@ -1834,7 +1834,7 @@ namespace cxxopts
       {
         if (o.has_implicit)
         {
-          result += " [=" + arg + "(=" + toLocalString(o.implicit_value) + ")]";
+          result += " " + arg + "(=" + toLocalString(o.implicit_value) + ")";
```

Comment:
The other changes look fine to me, but this one is inside cxxopts so your change may be overwritten if we update cxxopts at some point.  
What about removing it completely all the text in the brackets by piping `sed "s/\[.*\]//g"` just after `echo $f3dhelp`?